tests/system/src/test/java/test/javafx/scene/text/TestHandlingBigGlyphID_Win7_and_Win10.java

```diff
@@ -0,0 +1,38 @@
+import javafx.application.Application;
```

Comment:
Same comments for this test. Also, do they really need to be separate tests?

Reply:
For those who dislikes installing external font to Windows, I have created Test for Win10 which is not need to install font.
Test for Win7 needs to install font.